snuba/request/request_settings.py

```diff
@@ -88,3 +195,6 @@ def get_rate_limit_params(self) -> Sequence[RateLimitParameters]:
 
     def add_rate_limit(self, rate_limit_param: RateLimitParameters) -> None:
         pass
+
+    def get_sampling_config(self) -> SamplingConfig:
+        return AutoSamplingConfig()
```

Comment:
This is never turbo or sampled so it shouldn't this be `Fixed(1.0)`?

Reply:
ok, I just applied the same logic we have today.